src/scripts/main.js

```diff
@@ -1,7 +1,30 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const div = document.createElement('div');
+  const h2 = document.createElement('h2');
+  const p = document.createElement('p');
+  const body = document.querySelector('body');
+
+  div.style.position = 'absolute';
+  div.style.right = posRight + 'px';
+  div.style.top = posTop + 'px';
+
+  div.style.boxSizing = 'content-box';
```

Comment:
Do you really need to add it? Same with position absolute.

Reply:
I don't need position only but without content-box it will be broken